VulkanHppGenerator.cpp

```diff
@@ -1426,7 +1426,7 @@ void VulkanHppGenerator::readExtensionDisabledRequire(tinyxml2::XMLElement const
 {
   checkAttributes(getAttributes(element), element->GetLineNum(), {}, {});
   std::vector<tinyxml2::XMLElement const*> children = getChildElements(element);
-  checkElements(children, { "command", "enum", "type" });
+  checkElements(children, { "command", "comment", "enum", "type" });
```

Comment:
ISTM that any addition to the schema is likely to trigger this kind of problem - it appears that you've encoded a snapshot of the schema into the generator code.

Reply:
That's right, it's a snapshot of what I've encountered. I don't know how to handle stuff I haven't encountered, yet. So I try to make sure I'm getting notified when something unexpected appears in vk.xml that might need some additional handling.